Code/C++/Dijkstra.cpp

```diff
@@ -23,12 +23,17 @@ typedef vector<int> vi;
 typedef pair<int, int> pii;
 typedef vector<pii> vii;
 typedef pair<int, int> pii;
-vii *Graph;  // Graph -> Vector - Pair
-vi Distance; // It Stores the Distance of every other node from start.
+
+// Graph -> Vector - Pair
+vii *Graph;
+
+// It Stores the Distance of every other node from start.
+vi Distance;
 
 void Dijkstra(int start, int N)
 {
-    priority_queue<pii, vector<pii>, greater<pii>> Q; // min heap
+    // min heap
```

Comment:
use **4 tabs**  to indent the comment.

Reply:
isn't this one is 4 tabs indent?

Reply:
No, I have checked it in VS Code and it's 1 tab indent. And you also need to indent all the inline comments in the main section of the Dijkstra Algorithm by 4 tabs.

Reply:
Take the reference for indentation from [this code.](https://github.com/Algo-Phantoms/Algo-Tree/blob/main/Code/C%2B%2B/BFS.cpp)

Reply:
I am using Beautifier/prettier. So, No Issue !!